requirements_sparse.txt

```
@@ -0,0 +1 @@
+sparse>=0.15.4
```

Moving in `numba` from comment above.

```suggestion
sparse>=0.15.4
numba>=0.60.0
```

---

Does numba still need a cap on numpy? If tests are passing atm I guess it's a future problem?